### PR TITLE
Sales evidence widget filter

### PR DESCRIPTION
**Pull request by johnEthicalTechnology**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "google-maps-react": "^2.0.6",
     "jquery": "^3.5.1",
     "react": "^16.13.1",
+    "react-date-picker": "^8.0.1",
```
Comment (haydenf) [inline package.json]: remove this date picker, keep react-datepicker

File: src/services/crmDataService.ts [hunk 2]
```diff
@@ -15,24 +15,28 @@ async function getPageOfProperties (pageNumber: number) {
...
+        retrievedProperties.concat(thisPageResults),
+        entity
     )
 }
 
-export async function findMatchingProperties (searchParameters: SearchParametersType[]): Promise<{ matchedProperties: UnprocessedResultsFromCRM[], uniqueSearchRecords: string[] }> {
-    const matchingResults = await retrieveAllProperties(0, [])
+export async function findMatchingRecords (searchParameters: SearchParametersType[], entity: string): Promise<{ matchedProperties: UnprocessedResultsFromCRM[], uniqueSearchRecords: string[] }> {
+    const matchingResults = await retrieveAllRecords(0, [], entity)
+    console.log('matchingResults', matchingResults)
```
Comment (haydenf) [inline src/services/crmDataService.ts]: remove console.log

File: src/utils/leaseEvidenceSortAndFilter.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
+// TODO - tsconfig replace CommonJS with esnext
```
Comment (haydenf) [inline src/utils/leaseEvidenceSortAndFilter.ts]: Will it matter having the leases filter in this branch?

Comment (johnEthicalTechnology) [reply src/utils/leaseEvidenceSortAndFilter.ts]: I'll delete this. In `tsconfig.json` under `"compilerOptions"` I changed `"module": "esnext"` to `"module": "CommonJS"` so as to test the filter function I was using. [This](https://stackoverflow.com/questions/56546541/typescript-syntaxerror-unexpected-token) SO post helped me a bit.

**Pull request merged**